products/statement-generator/src/components-layout/Affirmation.tsx

```diff
@@ -38,6 +38,7 @@ const Affirmation = () => {
 
   const utilityClasses = useUtilityStyles({
     pageTheme: 'transparent',
+    justifyContent: 'flex-end',
```

Comment:
is the `justifyContent` still needed here?

Reply:
Hey! I removed `justifyContent` as it was not needed.